gammapy/spectrum/spectrum_fit.py

```diff
@@ -210,15 +199,17 @@ def _run_sherpa_fit(self):
         """Plain sherpa fit using the session object
         """
         from sherpa.astro import datastack
-        log.info("Starting SHERPA")
-        log.info(str(self))
+        from sherpa.utils.err import IdentifierErr
+        
+        # TODO: Make this optional
```

Comment:
You say I should not mute the logger from here, right?

The problem is that, since sherpa is imported in this function (and typically not in a user script) I only have access to the logger from here. I muted it because it is annoying when fitting O(100) simulated spectra. Do you propose to leave the logger as is and make users 'hack' it away (by importing sherpa) themselves?